### PR TITLE
Make Photobooth background image adjustable

### PR DESCRIPTION
**Pull request by andi34**
- Allow defining a background via admin panel

This also gives the possibility to define a livestream URL (e.g. `http://192.168.239.77:8081` if motion is used ) to use a livestream as background.

### CODE REVIEW

File: admin/index.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	<link rel="stylesheet" type="text/css" href="../resources/css/admin.css" />
 </head>
 <body class="deselect">
-<div id="wrapper" style="overflow-y: auto;">
+<div id="wrapper" style="overflow-y: auto; background-image: url('<?=$config['background_admin']?>');">
```
Comment (sualko) [inline admin/index.php]: I think it would be nice to use css vars like we do for the color schema. Therefore the html stays as clean as possible.

Comment (andi34) [reply admin/index.php]: Ok, will take a look next days

Comment (andi34) [reply admin/index.php]: Should I rename `_colors.scss` to `_theme.scss` and place it there ?

**Comment by andi34** [comment]
Only thing is the relative path is now throwing an error in the console.
Screenshot_2019-11-21_17-28-57

**Comment by andi34** [comment]
My fault. Fixed :)

**Comment by andi34** [comment]
Move out of lib/config.php and add to config/config.inc.php

It's complaining about the missing placeholder: https://github.com/andi34/photobooth/commit/8f9cfc0b78a29a151374a60fac9e5b5233f551db

**Comment by andi34** [comment]
Another issue now is the admin panel background uses the same background with the case styling. Have to fix that.

File: resources/sass/style.scss [hunk 2]
```diff
@@ -222,7 +218,7 @@ hr.small {
   color: #fff;
   width: 100%;
   height: 250px;
-  background: url(../img/bg.jpg) center center no-repeat;
+  background: $adminBackground center center no-repeat;
```
Comment (andi34) [inline resources/sass/style.scss]: $backgroundImage

**Comment by andi34** [comment]
https://github.com/andi34/photobooth/commit/d683c207b9b60dc4adcfaa33596d2b75d58bdaf0

I think this should fix it. Haven't tested, but will do in the evening.

**Comment by andi34** [comment]
Everything works as expected.

**Comment by sualko** [comment]
Had no time to test it, but looks good. We could rename the css variables so that they are more consistent, but this is only a minor issue.

**Comment by andi34** [comment]
Any good idea for the renaming?

**Comment by sualko** [comment]
After reading it again, it's fine.

**Pull request merged**